index.html

```diff
@@ -152,6 +152,33 @@ <h3>Skills</h3>
             </ul>
           </div>
         </section>  
+        <!--Get Started Section-->
+      <section id="get-started">
+        <img class="get-started-img" src="images/image_geometry_1.svg" alt="image_geometry_3">
+        <div class="container">
+          <h2>Get started</h2>
+          <p class="get-started-content">
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.
+        </p>
+        <form action="https://formspree.io/f/mlezegyq" method="POST">
+            <input type="text" id="name" name="user_name" maxlength="30" placeholder="Your name" required/>
+            <input type="email" id="email" name="user_email" placeholder="Your email" required/>
```

Comment:
- [ ] You've implemented the features of the form well. Please ensure you are following the Figma design as closely as possible. If you notice the font you are using does not match the Figma
## Your Design Font
![image](https://user-images.githubusercontent.com/18730106/149022811-794427de-ecb0-4373-8738-150dff7292b7.png)

## Figma Design Font
![image](https://user-images.githubusercontent.com/18730106/149022838-f2d67caa-67f5-477e-a28b-9eae36ae14a4.png)

The correct font to use is on the `input`s `Poppins`. you are currently using `Tahoma`.